care/emr/resources/observation_definition/spec.py

```diff
@@ -45,16 +46,55 @@ def validate_question_type(question_type):
     return question_type
 
 
+class RangeSpec(BaseModel):
+    interpretation: str
+    min: float | None = None
+    max: float | None = None
+    unit: str | None = None
+    text: str | None = None
+    values: list[str | float | int] | None = None
+
+    @model_validator(mode="after")
+    def validate_range(self):
+        if self.values is not None:
+            if self.min is not None or self.max is not None:
+                raise ValueError(
+                    "Cannot mix 'values' (for categorical data like positive/negative) with 'min' or 'max' (for numeric ranges). Choose one type."
+                )
+        elif self.min is None and self.max is None:
```

Comment:
```suggestion
        elif self.min is None or self.max is None:
```
this should be or right?